doc/en/components/grids/_shared/advanced-filtering.md

```diff
@@ -56,6 +65,22 @@ constructor() {
     grid.data = this.data
 }
 ```
+
+<!-- ComponentStart: TreeGrid -->
+```html
+<igc-tree-grid id="grid" auto-generate="true" allow-advanced-filtering="true">
+    <igc-grid-toolbar></igc-grid-toolbar>
+</igc-tree-grid>
+```
+<!-- ComponentEnd: TreeGrid -->
+
+```ts
+constructor() {
```

Comment:
This snippet get's already included in WC page from the above paragraph from line 47, causing it to duplicate